test/hotspot/jtreg/serviceability/jvmti/RedefineClasses/RedefineRetransform/libRedefineRetransform.cpp

```diff
@@ -45,6 +43,12 @@ static bool isTestClass(const char* name) {
     return name != nullptr && strcmp(name, testClassName) == 0;
 }
 
+/*
+ * Helper class for data exchange between RedefineClasses/RetransformClasses and
+ * ClassFileLoadHook callback (saves class bytes passed to CFLH,
+ * allows to set new class bytes to return from CFLH).
```

Comment:
typo: s/allows to set new/allows setting new/

Reply:
Done